specification/purview/data-plane/Azure.Analytics.Purview.Account/preview/2019-11-01-preview/account.json

```diff
@@ -418,6 +418,386 @@
         }
       }
     },
```

Comment:
pl work with the breaking changes reviewers to address the breaking changes before ARM can review this

Reply:
Is this enough? Created work item for review https://msazure.visualstudio.com/One/_workitems/edit/14018987